lib/smart_answer_flows/landlord-immigration-check.rb

```diff
@@ -87,10 +87,13 @@ def define
 
       #Q3a
       multiple_choice :what_nationality? do
+        option "british-or-irish"
```

Comment:
If you prefer to keep them separated please:

1. Clarify in the commit notes that the regression tests will be failing at this point and that they will be fixed in next commit
2. In the next commit, please remove the word `preparatory` in the commit title. It is just a artefacts regeneration. I would use: `Regenerate artefacts for inmigration-check` (I think this was the approach we used in the past)

In any case, I would try to have all commits in green state if possible.